spaces/S000088/properties/P000198.md

```diff
@@ -0,0 +1,9 @@
+---
+space: S000088
+property: P000198
+value: true
+---
+
+The space admits a closed cover consisting of two singletons and
```

Comment:
```suggestion
$X$ is the union of two singletons and
```
("closed cover" was unnecessary.  A countable union of subspaces with the property is all that's needed.)